lib/plug/templates/debugger.md.eex

```diff
@@ -0,0 +1,25 @@
+# <%= @title %> at <%= method(@conn) %> <%= @conn.request_path %>
+
+Exception:
+
+    <%= String.replace(@formatted, "\n", "\n    ") %>
+
+## Connection details
+
+### Params
+
+    <%= inspect(@params) %>
+
+### Request info
+
+  * URI: <%= url(@conn) %>
+  * Query string: <%= @conn.query_string %>
+  * Peer: <%= peer(@conn) %>
+
+### Headers
+    <%= for {key, value} <- Enum.sort(@conn.req_headers) do %>
+      <%= key %>: <%= value %><% end %>
```

Comment:
Nope, this was a mistake. I wanted to use 4 spaces (fixed it in the latest commit, I'm not really used to EEx 😄) because we get the code style if someone runs this through a markdown renderer, that was my target. Wdyt?